src/components/utils/guid.ts

```diff
@@ -0,0 +1,14 @@
+function generateId(counts: number[]): string {
```

Comment:
Does this need a test/spec file? Could be handled in a follow-up issue to add tests for utils.

Reply:
copied and pasted this from calcite repo as an interim solution - need to find out how both repos can properly share these utils - so we should only need to add a test once. Assuming this eventually goes in another repo - we can add the test there.